### PR TITLE
Fix #5019 - conda install pre_commit not found [skip ci]

### LINKED ISSUE #5019: [DOC] conda install pre_commit not found
## Report incorrect documentation

**Location of incorrect documentation**
https://github.com/rapidsai/cudf/blob/branch-0.14/CONTRIBUTING.md#setting-up-your-build-environment

**Describe the problems or issues found in the documentation**
When running  `conda install pre_commit`, conda does not find the package.

**Steps taken to verify documentation is incorrect**
I have googled the error message displayed when not finding the package.

**Suggested fix for documentation**
Use `conda install pre_commit -c conda-forge` instead.

---

## Report needed documentation

**Report needed documentation**
A clear and concise description of what documentation you believe it is needed and why.

**Describe the documentation you'd like**

**Steps taken to search for needed documentation**
List any steps you have taken:

### PR DESCRIPTION
**Pull request by miguelusque**
Hi!

When following the setting up environment guide (https://github.com/rapidsai/cudf/blob/branch-0.14/CONTRIBUTING.md#setting-up-your-build-environment), pre_commit package cannot be found.

I have added the channel conda-forge in order to help its resolution.

Hope it helps!

Regards,
Miguel

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by miguelusque** [comment]
rerun tests

**Comment by miguelusque** [comment]
Hi @kkraus14 ,

I have tried to update a doc, also mentioning the change in CHANGELOG file.

It passes some build test, but it fails in 3 of them.

I am not sure how to make it pass all the test (it is my first time submitting a PR in cuDF).

Could you please help me with the above?

Thanks!

Regards,
Miguel

**Comment by kkraus14** [comment]
@miguelusque there's a known issue with CI right now that is being triaged. I'll trigger a rerun when it's resolved. Thanks!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ to automatically run `isort`, `Black`, and `flake8` when you make a git commit.
 This can be done by installing `pre-commit` via `conda` or `pip`:
 
 ```bash
-conda install pre_commit
+conda install pre_commit -c conda-forge
```
Comment (kkraus14) [inline CONTRIBUTING.md:97]:
```suggestion
conda install -c conda-forge pre_commit
```

**Pull request merged**